src/full/Agda/Syntax/Concrete/Operators.hs

```diff
@@ -54,7 +53,6 @@ import Agda.Utils.Either
 import Agda.Utils.Pretty
 import Agda.Utils.List
 import Agda.Utils.List1 (List1, pattern (:|))
-import qualified Agda.Utils.List1 as List1
```

Comment:
The two imports are a fixed pattern and should not separated. Confer:
```haskell
import Data.Map (Map)
import qualified Data.Map as Map
```
Even if it happens that only one of these is actually needed, having the package of both saves the developer from worrying about whether the other half is present when working on the code.
I can't see any drawback of not having the package together.

Reply:
Agreed.